global_body_planner/include/global_body_planner/fast_global_motion_planner.h

```diff
@@ -97,10 +92,9 @@ class RRTConnectClass : public RRTClass {
    * @param[out] action_sequence The sequence of actions in the final path
    * @return Boolean for success of the planner
    */
-  int runRRTConnect(const PlannerConfig &planner_config, State s_start,
-                    State s_goal, std::vector<State> &state_sequence,
-                    std::vector<Action> &action_sequence,
-                    ros::Publisher &tree_pub);
+  int findPlan(const PlannerConfig &planner_config, State s_start, State s_goal,
```

Comment:
should start and goal states be passed by reference?

Reply:
Yes, although that will be a future PR into devel. This PR should not include any new changes to keep main and devel conflict-free.